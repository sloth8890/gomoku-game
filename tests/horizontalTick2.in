place A1
place B11
place E1
place B18
place B1
place E5
place C1
place D8
place D1
