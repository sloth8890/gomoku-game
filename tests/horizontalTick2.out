Black wins!
A1B11E1B18B1E5C1D8D1
Thank you for playing!
